src/storage/CookiesStorage.js

```diff
@@ -127,7 +127,17 @@ class CookiesStorage {
   }
 
   /** @override */
-  load(key, defaultValue) {}
+  load(key, defaultValue) {
+    if (!document.cookie.includes(`${key}=`)) {
+      return defaultValue;
+    }
+
+    const cookieValueJSON = document.cookie.split('; ')
+      .find((row) => row.startsWith(`${this.settings_['prefix']}${key}`))
+      .split('=')[1];
```

Comment:
Since the value of the cookie might contain an "=" as well, you'll want to return all the remaining values in that case.

Example cookie:
```
"my_base64_value=bGV0J3Mgc2VlIGlmIGFueW9uZSBkZWNvZGVzIHRoaXM="
```